mention-bot.js

```diff
@@ -301,7 +328,8 @@ function guessOwners(
     .filter(function(owner) {
       return config.userBlacklist.indexOf(owner) === -1;
     })
-    .slice(0, 3);
+    .slice(0, config.maxReviewers)
+    .concat(defaultOwners);
```

Comment:
This may mention the user twice. You probably want to remove the defaultOwners from the list with a filter before-hand.